security/xca/PortFile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+
+name                        xca
+version                     2.1.2
+revision                    0
+
+categories                  security crypto
+platforms                   darwin
+license                     xca
+
+maintainers                 {@lhaeger} openmaintainer
+
+description                 X Certificate and Key management
+long_description            XCA is a CA & PKI management tool that supports creating and managing X.509 certificates, \
+                            certificate requests, RSA, DSA and EC private keys, Smartcards and CRLs.
+homepage                    https://www.hohnstaedt.de/xca/
+
+master_sites                https://github.com/chris2511/xca/releases/download/RELEASE.${version}/
+distname                    ${name}-${version}
+
+checksums                   rmd160  16717ab6eb2cd33599d9606b7908551ad01bbff3 \
+                            sha256  fc845470a02b0b4534b46590be307f784662071fc412fdcad605c3bce901fe05 \
+                            size    1086563
+
+depends_lib                 port:qt5-qtbase \
+                            port:qt5-qtmacextras \
+                            port:qt5-qttranslations \
+                            port:qt5-qtdeclarative \
+                            port:qt5-qtmultimedia \
+                            port:qt5-qtsvg \
+                            port:qt5-qttools \
+                            port:qt5-sqlite-plugin \
```

Comment:
You will need to use the qt5 portgroup here instead. This will fix the builds on older macOS versions.

Reply:
Sounds good, thanks again.

Reply:
So, is there any documentation on the qt5 portgroup? It's not listed on https://guide.macports.org/chunked/reference.portgroup.html and apart from searching other portfiles, I cannot find any details.

Reply:
You will have to look at other ports that use it. For example, sqlitebrowser.